content/_code-samples/escrow/py/finish_escrow.py

```diff
@@ -0,0 +1,41 @@
+from xrpl.clients import JsonRpcClient
+from xrpl.models import EscrowFinish
+from xrpl.transaction import (safe_sign_and_autofill_transaction,
+                              send_reliable_submission)
+from xrpl.wallet import Wallet, generate_faucet_wallet
+
+client = JsonRpcClient("https://s.altnet.rippletest.net:51234") # connect to the testnetwork
+
+# Complete an escrow
+# cannot be called until the finish time is reached
+
+
+escrow_creator = generate_faucet_wallet(client=client).classic_address # ecsrow creator/ sender address
+escrow_seq = 27641268 # escrow sequence
+
+# optional fields
+
+# cryptic condition that must be met before escrow can be completed, passed on escrow creation
```

Comment:
I believe the technical term is Crypto Condition instead of `cryptic condition` - https://xrpl.org/known-amendments.html#cryptoconditions